templates/design40_webpages/wh/create_assembly_chargenumbers.html

```diff
@@ -35,7 +38,7 @@ <h1>[% HTML.escape(title) %]</h1>
       [% FOREACH part_id = stocked_by_parts_id.keys.sort -%]
         [% NEXT UNLESS stocked_by_parts_id.$part_id.size > 1 %]
         <tr class="listheading">
-          <td colspan="6">[% 'needed qty' | $T8 %]</td>
+          <td colspan="7">[% 'needed qty' | $T8 %]</td>
```

Comment:
Das colspan muss auch in Abhängigkeit zu INSTANCE_CONF.get_show_bestbefore geändert werden, sonst stimmt es nicht, wenn MHD aus ist.
Gleiches im alten Design.